app/assets/stylesheets/checkin.scss

```diff
@@ -1,9 +1,23 @@
 @import "colors";
 
 .disable-field-set {
-  background: $gray;
+  background: #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.